### ISSUE TITLE
"FailedToStartServiceHealthcheck... bind: address already in use" on dual-stack cluster

### ISSUE THREAD

**Issue by h3mmy** [h3mmy/bloopySphere#6390]
# Details

NOTE: This is not a new issue. I've been waiting on an upstream fix. Just want to document it for my future self.
**What steps did you take and what happened:**

Check events. Routinely see events like this one:

```
53m         Warning   FailedToStartServiceHealthcheck   service/lldap-ldaps     node slowpoke failed to start healthcheck "auth/lldap-ldaps" on port 32056: listen tcp 0.0.0.0:32056: bind: address already in use
```

**What did you expect to happen:**

No `FailedToStartServiceHealthcheck` events

**Anything else you would like to add:**

**Additional Information:**

This is been there for years, but it is annoying. The root cause is the way kube-proxy handles dual-stack networks and is already reported in the kubernetes project at kubernetes/kubernetes#114702

For ref a `sudo netstat -tulnp | grep LISTEN` shows this for port 32056

```
tcp6       0      0 :::32056                :::*                    LISTEN      873/k3s server
```

**Comment by h3mmy** [comment]
I suppose I can just get to acquiring more nodes and run talos. That might fix ~~me~~ _this issue_, but it's currently out of budget
